.github/workflows/osx.yml

```
@@ -0,0 +1,50 @@
+name: OSX-build
+
+on: [push]
+
+jobs:
+  osx-clang-shared:
+
+    name: OSX build shared libs
+    runs-on: [macos-latest]
+
+    steps:
+    - uses: actions/checkout@v1
```

```suggestion
    - uses: actions/checkout@v2
```

checkout@v2 which only pulls the current branch is faster than checkout@v1